src/include/daos/btree.h

```diff
@@ -33,6 +33,14 @@
 #include <daos_types.h>
 #include <daos/mem.h>
 
+/**
+ * With Checksum enabled, direct key should still have a hash
+ */
+struct btr_hash_direct {
+	umem_off_t		rec_node; /* direct key */
+	char		rec_hkey[0]; /* hashed key */
```

Comment:
is the spacing off here or is it just github?

Reply:
fixed